akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -72,6 +72,9 @@ sealed trait HttpEntity extends jm.HttpEntity {
       .via(new akka.http.impl.util.ToStrict(timeout, contentType))
       .runWith(Sink.head)
 
+  def discardEntityBytes(mat: Materializer): HttpMessage.DiscardedEntity =
```

Comment:
In that case, I prefer  `request.discardEntityBytes()` over the member access for call